src/scripts/modules/featured-books/featured-books.coffee

```diff
@@ -45,7 +42,10 @@ define (require) ->
     onAfterRender: () ->
       $('.show-more-less .more').on('click', @readMore)
       $('.show-more-less .less').on('click', @readLess)
-      $(window).resize(@debouncedShaveBookDescriptionsAfterImagesLoaded)
-
-    onBeforeClose: () ->
-      $(window).off('resize', @debouncedShaveBookDescriptionsAfterImagesLoaded)
+      $('.description').each (i, desc) ->
+        $(desc).css('max-height', '100%')
+        if $(desc).height() <= 60
```

Comment:
Won't this be a problem if it is checked before the images are loaded?
They tend to compress the text...
Might be fine, but have to test while simulating a slow connection with the dev console.

Reply:
I've tested this with `Regular 3G` connection and it looks fine.
btw. When I was trying to simulate `Good 2G` nothing was loaded at all. Some scripts are timeouting.